plugin/src/main/java/at/helpch/chatchat/config/holders/MessagesHolder.java

```diff
@@ -33,6 +33,8 @@ public final class MessagesHolder {
     private Component socialSpyDisabled = text("Social spy disabled", RED);
     private Component ignoredPlayer = text("Successfully ignored <player>.", GREEN);
     private Component unignoredPlayer = text("Successfully un-ignored <player>.", GREEN);
+    private Component cantMessageIgnoredPlayer = text("You cannot message a player who you ignore.", RED);
+    private Component currentlyIgnored = text("You are ignored by <player>.", RED);
```

Comment:
We don't really need a <player> placeholder here and I was thinking the default message should be just "You cannot message this user!". So basically don't directly tell them they're ignored but also let them know the message can't be sent. If server owners feel like they want to tell the sender that they're ignored by the recipient, then sure they can change it themselves.